lib/zipkin-tracer/trace.rb

```diff
@@ -0,0 +1,67 @@
+require 'finagle-thrift'
+require 'finagle-thrift/trace'
+
+module Trace
+  class Span
+    def to_h
+      {
+        name: @name,
+        traceId: @span_id.trace_id.to_s,
+        id: @span_id.span_id.to_s,
+        parentId: @span_id.parent_id.nil? ? nil : @span_id.parent_id.to_s,
```

Comment:
saw that in the json api responses when doing GETs but I assumed it was something the server computed to make things easier for the front end side.
none of the tracers do that and it works without sending these so how about it keeping it as is, at least for now?
I'm open to change this if there's a compelling argument in favor of doing this in the client VS the server.